source/contracts/reporting/Universe.sol

```diff
@@ -56,6 +57,13 @@ contract Universe is DelegationTarget, ITyped, Initializable, IUniverse {
         require(isContainerForMarket(IMarket(msg.sender)));
         forkingMarket = IMarket(msg.sender);
         forkEndTime = block.timestamp + Reporting.forkDurationSeconds();
+        // We pre calculate the amount of REP needed to determine a winner early in a fork. We assume maximum possible fork inflation in every fork so this is hard to achieve with every subsequent fork and may become impossible in some universes.
+        if (parentUniverse != IUniverse(0)) {
+            uint256 _previousForkReputationGoal = parentUniverse.getForkReputationGoal();
+            forkReputationGoal = _previousForkReputationGoal + (_previousForkReputationGoal / Reporting.forkMigrationPercentageBonusDivisor());
+        } else {
+            forkReputationGoal = reputationToken.totalSupply() / Reporting.forkRepMigrationVictoryDivisor();
```

Comment:
I think this should be hard-coded to 11M, not `reputationToken.totalSupply()`.  When Augur "launches", the universe will start out with a total supply of 0 and increase as people migrate REP in from the current REP token.  It is possible that the first fork occurs before everyone migrates their REP, in which case the first fork would suffer from the same problem we are trying to avoid. with this calculation.

Reply:
Got it. Will add a comment to that effect too.